src/record/export/repositories/stringifiers/csvStringifier/index.ts

```diff
@@ -0,0 +1,50 @@
+import type { LocalRecord } from "../../../types/record";
+import type { CsvRow } from "../../../../../kintone/types";
+import type { RecordSchema } from "../../../types/schema";
+
+import stringify from "csv-stringify";
+
+import { convertRecord, recordReader } from "./record";
+import { LINE_BREAK, SEPARATOR } from "./constants";
+import { buildHeaderFields } from "./header";
+import type { Stringifier } from "../index";
+
+export class CsvStringifier implements Stringifier {
+  private readonly csvStringifier: stringify.Stringifier;
+  private readonly schema: RecordSchema;
+  private readonly useLocalFilePath: boolean;
+  constructor(schema: RecordSchema, useLocalFilePath: boolean) {
+    this.schema = schema;
+    this.useLocalFilePath = useLocalFilePath;
+
+    const headerFields = buildHeaderFields(schema);
+    this.csvStringifier = stringify({
+      columns: headerFields,
+      header: true,
+      delimiter: SEPARATOR,
+      record_delimiter: LINE_BREAK,
+      quoted_match: /^(?!\*$).*$/,
+      quoted_empty: false,
+    });
+  }
+
+  async stringify(records: LocalRecord[]): Promise<string> {
```

Comment:
Before this PR, cli-kintone only header, but cli-kintone on current branch outputs nothing when empty app.  

```console
# before
$ cli-kintone record export --base-url https://****.kintone.com --app * --username ******** --password ********
"Record_number","Number","Date","Time","Date_and_time","Created_by","Created_datetime","Updated_by","Updated_datetime"

# after
$ cli-kintone record export --base-url https://****.kintone.com --app * --username ******** --password ********
```
The cli-kintone crashes here on this function with no errors and no outputs.

Reply:
@ueokande 
Thank you for your review!
Exactly, it was the bug of the stringifier.
I fixed it and added a test with no record case.

https://github.com/kintone/cli-kintone/blob/feat/improve-memory-usage-of-export/src/record/export/repositories/stringifiers/csvStringifier/__tests__/fixtures/index/withNoRecord/index.ts

Reply:
@ueokande 
Now it might work correctly.
Please check the behavior again, please.

```
$ node ./cli.js record export --app $APP_ID 
"Record_number","Created_by","Created_datetime","Updated_by","Updated_datetime"
```